source/dashboards/dashboards.markdown

```diff
@@ -55,12 +59,26 @@ The **Areas** dashboard is prepopulated by default and shows your {% term entiti
 Screenshot of the Areas default dashboard.
 </p>
 
-After you [added the dashboard](#creating-a-new-dashboard), you can edit the **Areas** dashboard:
+#### Limitations of the Areas dashboard
 
-1. In the top-right corner, select the {% icon "mdi:pencil" %} icon.
-2. You can show or hide sections, rearrange the content, or [add badges](/dashboards/badges/).
+1. Not all devices are added to the **Areas** dashboard. Only a few sensors are displayed automatically. Such as temperature and humidity of the area, and door, window, and gate binary sensors.
+2. Certain entity types and helpers are omitted from the **Areas** dashboard. For example, configuration and diagnostic {% term entities %} do not appear, even if they have been assigned to an area.
```

Comment:
```suggestion
2. Certain entity domains and helpers are omitted from the **Areas** dashboard. For example, configuration and diagnostic {% term entities %} do not appear, even if they have been assigned to an area.
```

Should it be domains instead of types?